docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Teleport with GitHub.
+description: How to use SSH key extensions with Teleport and GitHub.
```

Comment:
What can users do with Teleport, GitHub, and SSH key extensions? Right now, "use" is pretty vague to me.